broker/httpd/api.go

```diff
@@ -42,10 +45,23 @@ func StartServer(staticConfig *staticconfiguration.StaticConfiguration, brokers
 		logger:       logger,
 		staticConfig: staticConfig,
 	}
+	// load templates
+	server.htmlTemplate = template.New("main")
+	/// Load the oter built in templates
+	extraTemplates := []string{footerTemplateText, consoleAccessTemplateText, headerTemplateText}
+	for _, templateString := range extraTemplates {
+		_, err = server.htmlTemplate.Parse(templateString)
+		if err != nil {
+			return nil, err
+		}
+	}
+
 	http.HandleFunc("/", server.rootHandler)
 	http.HandleFunc("/status", server.statusHandler)
 	serviceMux := http.NewServeMux()
-	serviceMux.HandleFunc("/", server.rootHandler)
+	serviceMux.HandleFunc("/", server.consoleAccessHandler)
+	serviceMux.HandleFunc("/static/", staticHandler)
```

Comment:
Why the trailing slash?

Reply:
I like terminating slashes for clarity.

Reply:
Hm. OK for HTTP URLs, I guess. Can you please make this consistent, then?